tensorflow/python/keras/saving/hdf5_format.py

```diff
@@ -52,8 +53,21 @@
     "tensorflow.python.keras.engine.sequential")
 # pylint:enable=g-inconsistent-quotes
 
+# create lock file
+def create_lockfile(filepath):
+  lockfile_path = f"{filepath}.lock"
 
-def save_model_to_hdf5(model, filepath, overwrite=True, include_optimizer=True):
+  f = gfile.GFile(lockfile_path, 'w')
+  f.write(f"{os.getpid()}")
```

Comment:
I think f-strings are only available since Python 3.6. But we need to still support 3.5.